lib/puppet/file_serving/configuration.rb

```diff
@@ -82,6 +83,8 @@ def mk_default_mounts
     @mounts["plugins"].allow('*') if @mounts["plugins"].empty?
     @mounts["pluginfacts"] ||= Mount::PluginFacts.new("pluginfacts")
     @mounts["pluginfacts"].allow('*') if @mounts["pluginfacts"].empty?
+    @mounts["tasks"] ||= Mount::Tasks.new("tasks")
+    @mounts["tasks"].allow('*') if @mounts["tasks"].empty?
```

Comment:
I think we also need to add a default rule for the new endpoint to `conf/auth.conf` and `Puppet::Network::DefaultAuthProvider.default_acl`. I think the former is used in packaging (maybe?). The latter is used if auth.conf is missing and we're accessing the ruby endpoint directly (not from puppetserver).